frontend/src/theme/theme.tsx

```diff
@@ -8,6 +8,12 @@ declare module '@mui/material/Chip' {
   }
 }
 
+declare module '@mui/material/styles' {
+  interface Palette {
+    paper: Palette['primary'];
+  }
+}
+
```

Comment:
@yishain11 What's the reason for this code?

Reply:
this is the button color issue I talked to you about, that I did with Aviran. The idea is to use the theme in order to color the buttons unless someone adds a color of their own.

Reply:
I fixed it and added another color, so you can see the point of this code. basicly, its to add another option to the allowed colors so typescript wont crash everything